UPGRADE-API-1.13.md

```diff
@@ -63,166 +129,122 @@
+   API continues to support the explicit sending of the `locale` field, it is essential that this explicitly sent locale
+   matches the key in the translation array. In cases of a mismatch between the key and an explicitly sent locale, the
+   API will respond with a `Sylius\Bundle\ApiBundle\Exception\TranslationLocaleMismatchException`.
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\SendOrderConfirmationHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private OrderRepositoryInterface $orderRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\OrderEmailManagerInterface $orderEmailManager,
-        ) {
-        }
-    ```
+1. Disabled product and taxon editing at `/admin/product-taxons/{id}` operation to improve data integrity. To modify a
+   productTaxon, remove the existing association and create a new one.
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\SendShipmentConfirmationEmailHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private ShipmentRepositoryInterface $shipmentRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ShipmentEmailManagerInterface $shipmentEmailManager,
-        ) {
-        }
-    ```
+1. The keys for adjustment endpoints' responses have been changed from `order_item` to `orderItem` and `order_item_unit`
+   to `orderItemUnit`.
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\SendContactRequestHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private ChannelRepositoryInterface $channelRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ContactEmailManagerInterface $contactEmailManager,
-        ) {
-        }
-    ```
+### Miscellaneous changes
 
-    `Sylius\Bundle\CoreBundle\MessageHandler\Admin\Account\SendResetPasswordEmailHandler`: 
-   ```diff
-        public function __construct(
-            private UserRepositoryInterface $shopUserRepository,
-    -       private SenderInterface $emailSender,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ResetPasswordEmailManagerInterface $resetPasswordEmailManager,
-        ) {
-        }
-    ```
+1. All the `:read` serialization groups are now split to `index` and `show`.
+   By this change, the `:read` serialization group is now deprecated and will no more used in the future.
+   There is a BC layer that will allow you to use the `:read` serialization
+   group `Sylius\Bundle\ApiBundle\SerializerContextBuilder\ReadOperationContextBuilder` by adding the `read`
+   serialization group to your context.
+   Inside of this service there are 2 configurable parameters `$skipAddingReadGroup` and `$skipAddingIndexAndShowGroups`
+   that will allow you to skip adding the chosen serialization group to your context.
```

Comment:
```suggestion
```
The same thing is explained right below, just with a config file and not PHP variables